src/docs/introduction/why-stencil.md

```diff
@@ -4,39 +4,38 @@ description: Stencil has a number of add-ons that you can use with the build pro
 url: /docs/introduction
 contributors:
   - jthoms1
+  - splitinfinities
+  - a-giuliano
 ---
 
-# Stencil: A Compiler for Web Components and High Performance Web Apps
+# Overview
 
-Stencil is a compiler that generates Web Components (more specifically, Custom Elements) and builds high performance web apps. Stencil combines the best concepts of the most popular frameworks into a simple build-time tool.
+## Stencil: A Web Components Compiler
 
-Stencil takes features such as
+Stencil is a compiler that generates Web Components (more specifically, Custom Elements). Stencil combines the best concepts of the most popular frameworks into a simple build-time tool.
 
-- Virtual DOM
-- Async rendering (inspired by React Fiber)
-- Reactive data-binding
-- TypeScript
-- JSX
-- Static Site Generation (SSG)
+Stencil uses TypeScript, JSX, and CSS to create standards-based Web Components that can be used to craft high quality component libraries, design systems, and applications.
 
-and then generates standards-based Web Components and web apps with these features baked in.
+Stencil generates standards-compliant Web Components that can work with popular frameworks right out of the box. In addition, Stencil can be used to generate framework native components that can be used just like any other components in your framework of choice. Stencil accomplishes this by wrapping your Web Components via Stencil's Output Target feature.
```

Comment:
Are we officially using 'Output Target' as the verbiage here to describe the Angular/React/Vue wrappers? IDK when the plan to work that in ('Output Target' -> 'Framework Wrapper') was, feel free to leave in if you think that's best saved for another day. IE to keep consistency with the rest of the site, I think it'd be fine to leave this as-is.

Reply:
Yeah I left this as 'Output Target' for consistency of the site. We started to use the framework wrapper verbiage in the react and angular docs, but I kept it as Output Target here so users can at least understand what the 'Output Target" section is when the navigate to it. Once there we can introduce the new verbiage. I'd like to completely transition to "framework wrappers" once the code is aligned with that terminology.